internal/cmd/miniooni/auth.txt

```diff
@@ -0,0 +1 @@
+secret
```

Comment:
Why do we want to ignore this specific file? Is this something you committed for testing and forgot to uncommit?

Reply:
oops, yep it should not be there